pkg/lib/robots/robots_txt.go

```diff
@@ -0,0 +1,23 @@
+package robots
```

Comment:
I suggest we put this in httputil. And make this handler like `HealthCheckHandler`.